WordPress/ruleset.xml

```diff
@@ -7,14 +7,10 @@
 	<rule ref="WordPress-Core"/>
 	<rule ref="WordPress-Docs"/>
 	<rule ref="WordPress-Extra"/>
-	<rule ref="WordPress-VIP"/>
 
 	<rule ref="WordPress.PHP.DiscouragedPHPFunctions">
-		<!-- From "VIP": The obfuscation group is excluded as there are plenty of legitimate uses for the base64 functions. -->
```

Comment:
@tomjn This is the part which is causing the parse error in the XML file.
The whole block should be put back, i.e. line 12 - 17, not just line 12.

And as I've not seen a passing build yet and you seemed to have trouble getting the XML to validate before, the principle against which it is validated is quite simple:
* Valid XML
* Use tab indents (not spaces)
* No trailing whitespace anywhere.

Reply:
argh I just realised this is the WP ruleset in `develop`:

```xml
	<rule ref="WordPress.PHP.DiscouragedPHPFunctions">
		<!-- From "VIP": The obfuscation group is excluded as there are plenty of legitimate uses for the base64 functions. -->
		<properties>
			<property name="exclude" type="array" value="obfuscation"/>
		</properties>
	</rule>
```

and here it is locally:

```xml
	<rule ref="WordPress-Extra"/>

	<rule ref="WordPress.PHP.DiscouragedPHPFunctions">

	<exclude name="WordPress.VIP"/>
```

It's the missing closing slash

Reply:
> It's the missing closing slash

No, the complete `<properties>` block should be put back.